### PR TITLE
feat(deps): upgrades lerna to 6.x.x

### PR DESCRIPTION
**Pull request by pavankjadda**
### PR Checklist

- [x] Brief description of changes
- [ ] Links to any relevant issues
- [ ] Required status checks are passing
- [ ] User cases if changes impact the user's experience
- [x] `@mention` a maintainer to request a review

### Updates `lerna` version to 5.1.4

1. `Lerna` is now maintained by [NX team at Nrwl](https://github.com/lerna/lerna/issues/3121)
2. Version 5.1.4 includes bug fixes and performance improvements

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

**Comment by pavankjadda** [comment]
@sedghi please review this

**Comment by sedghi** [comment]
Thanks @pavankjadda for this PR.

Do you have any experience with the [nx](https://nx.dev/), and do you think that is a good option to replace Lerna now that it is dead (kind of...)

**Comment by pavankjadda** [comment]
> Thanks @pavankjadda for this PR.
> 
> 
> 
> Do you have any experience with the [nx](https://nx.dev/), and do you think that is a good option to replace Lerna now that it is dead (kind of...)

I do. We use it in one of our projects, it's really good. But when we upgrade Learn 5.x we don't have to use it, but we can. NX team took ownership of the project and will maintain Lerna from now on.

File: platform/viewer/package.json [hunk 1]
```diff
@@ -24,7 +25,7 @@
     "build:viewer:package": "yarn run build:package",
     "dev": "cross-env NODE_ENV=development webpack-dev-server --config .webpack/webpack.pwa.js",
     "dev:orthanc": "cross-env NODE_ENV=development PROXY_TARGET=/dicom-web PROXY_DOMAIN=http://localhost:8042 APP_CONFIG=config/docker_nginx-orthanc.js webpack-dev-server --config .webpack/webpack.pwa.js",
-    "dev:viewer": "yarn run dev",
+    "dev:viewer": "yarn run dev --port 5001",
```
Comment (sedghi) [inline platform/viewer/package.json:28]: Is it required in lerna 5.x to define port?

Comment (pavankjadda) [reply platform/viewer/package.json]: Not needed. I added as part of my local test because of 3000 conflict. Removed it in new commit

**Comment by pavankjadda** [comment]
@sedghi Do you have anymore questions about this? If not we can merge this.

**Comment by pavankjadda** [comment]
@sedghi As I said before, lerna and NX are two different projects maintained by same company. They are making very good improvements to Lerna. I think we can go ahead and use 6.5.x version. They also made upgrade painless.

**Comment by sedghi** [comment]
@pavankjadda Hey
We are pushing the v3-stable branch to master soon, I'm more interested in that upgrade than the v2. Have you tried it on the v3-stable?

**Comment by pavankjadda** [comment]
> @pavankjadda Hey We are pushing the v3-stable branch to master soon, I'm more interested in that upgrade than the v2. Have you tried it on the v3-stable?

I can create new PR of v3-stable. Would that be okay?

**Comment by sedghi** [comment]
yes indeed

**Comment by pavankjadda** [comment]
> yes indeed

Okay. Created https://github.com/OHIF/Viewers/pull/3215 on v3-stable

**Pull request closed without merging**